src/education.njk

```diff
@@ -4,7 +4,7 @@ nohero: true
 meta:
     title: "FlowFuse for Education"
     keywords: FlowFuse, FlowFuse for students, FlowFuse for educators, FlowFuse for educational institutions, FlowFuse for universities, FlowFuse for schools, FlowFuse for colleges, FlowFuse for learning, FlowFuse for teaching, FlowFuse for training, FlowFuse for research, FlowFuse for academic institutions, FlowFuse for educational organizations, FlowFuse for educational establishments, FlowFuse for educational bodies, FlowFuse for educational centers, FlowFuse for educational facilities, Node-RED for education, Node-RED education, Node-RED for universities, Node-RED for classroom, Node-RED for educators, Node-RED for universities, Node-RED for academic institutions
-description: "FlowFuse offers a comprehensive suite of tools and resources for educational institutions, students, and educators to accelerate learning, teaching, and research in Node-RED and industrial automation."
+description: "FlowFuse offers a comprehensive suite of tools and resources for accredited educational institutions to accelerate learning, teaching, and research in Node-RED and industrial automation. Instructors and educational administrators are invited to contact us for an educational license, which will supply you and your students with 5 free months of FlowFuse Cloud Starter."
```

Comment:
Hi @gstout52 this is the meta description, it's used for SEO and in this page it's not visible. I think this one was ok.

```suggestion
description: "FlowFuse offers a comprehensive suite of tools and resources for educational institutions, students, and educators to accelerate learning, teaching, and research in Node-RED and industrial automation."
```

Reply:
The text you need to update is on line 25. I didn't update it myself because it's not exactly like the meta description, and you might prefer to create a different version based on this one.